pkg/kubeapiserver/authorizer/reload.go

```diff
@@ -140,3 +159,120 @@ func (r *reloadableAuthorizerResolver) newForConfig(authzConfig *authzconfig.Aut
 
 	return union.New(authorizers...), union.NewRuleResolvers(ruleResolvers...), nil
 }
+
+// runReload starts checking the config file for changes and reloads the authorizer when it changes.
+// Blocks until ctx is complete.
+func (r *reloadableAuthorizerResolver) runReload(ctx context.Context) {
```

Comment:
Can we write one shared controller that handles the file watching and polling that we can re-use for authn/authz/kms?